elasticsearch/Dockerfile

```diff
@@ -8,18 +8,19 @@ USER 0
 
 ENV HOME=/opt/app-root/src \
   JAVA_VER=1.8.0 \
-  ES_VER=2.3.5 \
-  SG_VER=2.3.5.7 \
-  SG_SSL_VER=2.3.5.15 \
-  ES_CLOUD_K8S_VER=2.3.5 \
-  OSE_ES_VER=2.3.5.7 \
+  ES_VER=2.4.1 \
+  SG_VER=2.4.1.8 \
+  SG_SSL_VER=2.4.1.16 \
+  ES_CLOUD_K8S_VER=2.4.1 \
+  OSE_ES_VER=2.4.1.0 \
   ES_CONF=/usr/share/elasticsearch/config \
   INSTANCE_RAM=512G \
   NODE_QUORUM=1 \
   RECOVER_AFTER_NODES=1 \
   RECOVER_EXPECTED_NODES=1 \
   RECOVER_AFTER_TIME=5m \
-  PLUGIN_LOGLEVEL=INFO
+  PLUGIN_LOGLEVEL=INFO \
+  ES_JAVA_OPTS="-Dmapper.allow_dots_in_name=true"
```

Comment:
I think it should be possible, the say in [Enabling support for dots in field names](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/dots-in-names.html#_enabling_support_for_dots_in_field_names) that:

> (ES_JAVA_OPTS can be set in the /etc/default/elasticsearch file if you’re using the Debian package, or in the /etc/sysconfig/elasticsearch file if you’re using the RPM.)

If it can be set in `elasticsearch.yml` then I would say let's move it in that file so that we have all setting in one place.